src/viam/media/utils.py

```diff
@@ -0,0 +1,75 @@
+from array import array
```

Comment:
This file I think should be called `pil.py` within a `utils` directory IMO. Since this file only contains utils for PIL

Reply:
Are you suggesting to make a `src/viam/utils/` directory with `pil.py` in it? Would this directory also include the already existing`src/viam/utils.py`? That would be a bigger (and breaking) change as well, since so many places call from it.
I realized `src/viam/utils.py` has everything with no real organization; should I just put the contents of `pil.py` in there?